cron_job.js

```diff
@@ -28,39 +28,42 @@ var job = new cron.CronJob({
     timeZone: 'America/Anchorage'
 });
 
-
 function sendMessages() {
-    // instantiating this here so it'll run without an auth token in dev
-    var twilio_client = twilio(TWILIO_SID, TWILIO_AUTH_TOKEN)
-
     // get weather
     forecast.getLowTemps(function(err, data) {
         if (err) return rollbar.handleError(err)
+        var subscribers = db('subscribers').filter(sub => data[sub.zip] <= NOTIFICATION_TEMPERATURE)
 
-        db('subscribers').forEach(function(subscriber) {
-            console.log(subscriber)
-            if (data[subscriber.zip] <= NOTIFICATION_TEMPERATURE) {
-                twilio_client.sendMessage(
-                    {
-                        to: subscriber.phone,
-                        from: TWILIO_NUMBER,
-                        body: randomElement(message_text.NOTIFICATIONS),
-                    },
-                    function (err, response) {
-                        if (err) return rollbar.handleError(err)
-
-                        console.log(response)
-                    }
-                )
-            }
-        })
+        // loop over subscribers with delay
+        var inter = setInterval( gen => {
+            var next = gen.next()
+            if (next.done)  return clearInterval(inter)
+            var subscriber = next.value
+            module.exports.sendToTwilio(subscriber.phone, randomElement(message_text.NOTIFICATIONS))
+        }, DELAY, subscribers[Symbol.iterator]())
     })
 }
 
+function sendToTwilio(to, body){
```

Comment:
looks like this doesn't need to accept the body

Reply:
Thanks for catching that -- the `to:` is also wrong. I think it makes sense to pass the body in, so this function is just responsible for sending, but I'll change it to make it actually use the argument.

Reply:
I think it's fixed.